### PR TITLE
Merge master-next into master

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: mender/values.yaml [hunk 1]
```diff
@@ -85,7 +104,7 @@ deployments:
   image:
     registry: ""
     repository: ""
-    tag: mender-3.4.0
+    tag: mender-master
```
Comment (lluiscampos) [inline mender/values.yaml:107]: This should be 3.5.0

File: mender/values.yaml [hunk 2]
```diff
@@ -113,7 +132,7 @@ device_auth:
...
```
Comment (lluiscampos) [inline mender/values.yaml:135]: 3.5.0

File: mender/values.yaml [hunk 3]
```diff
@@ -150,7 +189,7 @@ gui:
   image:
     registry: docker.io
     repository: mendersoftware/gui
-    tag: mender-3.4.0
+    tag: mender-master
```
Comment (lluiscampos) [inline mender/values.yaml:192]: Here and everywhere below: 3.5.0

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -49,7 +49,7 @@ build:
 
 test:helm_chart_install:
   rules:
-    - if: '$CI_COMMIT_BRANCH == "master" || $RUN_HELM_CHART_INSTALL == "true"'
+    - if: '$CI_COMMIT_BRANCH == "master" || $CI_COMMIT_BRANCH == "master-next" || $RUN_HELM_CHART_INSTALL == "true"'
```
Comment (lluiscampos) [inline .gitlab-ci.yml:52]: This we can revert

**Review by lluiscampos — CHANGES_REQUESTED** [review]

File: .gitlab-ci.yml (hunk 4 above)
Comment (tranchitella) [reply .gitlab-ci.yml:52]: I'd keep it as it is, this is handy when we need a feature branch.

**Review by lluiscampos — APPROVED** [review]
Thank you

**Pull request merged**